### ISSUE TITLE
Add MCU data table to README

### ISSUE THREAD

**Issue by balotofi** [BIRDSOpenSource/BIRDS4-OBC#5]
TAKE INSPO FROM THIS REPO AND ADD A SIMILAR TABLE LIKE THE ONE BELOW

https://github.com/cpq/bare-metal-programming-guide?tab=readme-ov-file

https://github.com/nhivp/Awesome-Embedded?tab=readme-ov-file

![Image](https://github.com/BIRDSOpenSource/BIRDS4-OBC/assets/100206676/9fc19b9d-8db5-44c7-85c2-140ae86f1f10)

**Comment by Ziphozenkosimthombe** [comment]
hey so I must add the samethings that are in the image above?

**Comment by balotofi** [comment]
yeah so in the readme file, you need to create a table similar to the one above by finding the information in the documents in the repository

**Comment by Ziphozenkosimthombe** [comment]
so about finding the information in docs you mean the link that is in the description of the projects . or I must use the repositories that you have put above or there were just for reference ?.

**Comment by balotofi** [comment]
> so about finding the information in docs you mean the link that is in the description of the projects . or I must use the repositories that you have put above or there were just for reference ?.

You will find necessary information in the files in this repository. The links in this description are just for reference
